test/lib/builder/builder.js

```diff
@@ -277,6 +277,9 @@ test("Build application.g", (t) => {
 	const expectedPath = path.join("test", "expected", "build", "application.g", "dest");
 
 	return builder.build({
+		includedTasks: [
```

Comment:
Is there any integration test left that does not execute the generateResourceJson task now?

Reply:
distributed it more